signer/src/stacks/api.rs

```diff
@@ -957,8 +1006,14 @@ mod tests {
         second_mock.assert();
     }
 
+    #[test_case(|url| StacksClient::new(url, 20).unwrap(); "stacks-client")]
```

Comment:
P.S. we should use this kind of test even over on the bitcoin client to ensure that the logic behaves the same (i.e. that the base-client method is being called correctly)